.github/pull_request_template.md

```diff
@@ -13,8 +13,8 @@ Describe the original problem and the changes made on this PR.
 * JIRA: https://zendesk.atlassian.net/browse/AF-XXX
 
 ### Risks
-* [HIGH | medium | low] Does it work across browsers (including IE!)?
-* [HIGH | medium | low] Does it work in the different products (Support, Chat, Connect)?
```

Comment:
Are we planning on having apps in Connect at some point?

Reply:
Hi @token-cjg ,  You're right. Thanks for noticing. I realised this and changed `Connect` to `Guide` in the latest push.

Reply:
Guide is also not confirmed yet. So maybe leave it to support and chat for now.